From 86a503d91990920a1abd811f12c17fbe31c7a5c6 Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <robotux@celest.fr>
Date: Thu, 3 Jan 2013 16:46:23 +0100
Subject: Fix typos in string shown to the user.

s/authentification/authentication/ AuthPlugin.cpp
s/informations/information/ SystrayIcon.cpp
s/informations/information/ HelpDialog.ui
s/informations/information/ plugins/Languages/*/translation.ts
s/informations/information/ plugins/Themes/Oxygen/options.ui
s/informations/information/ plugins/Themes/Oxygen/interface.ui
s/informations/information/ plugins/Themes/Oxygen/Languages/*/translation.ts

Origin: vendor
Forwarded: no
Last-Update: 2013-01-03
---
 AuthPlugin.cpp                                    |    2 +-
 HelpDialog.ui                                     |    2 +-
 SystrayIcon.cpp                                   |    2 +-
 plugins/Languages/ar/translation.ts               |   10 +++++-----
 plugins/Languages/de/translation.ts               |   10 +++++-----
 plugins/Languages/el/translation.ts               |   10 +++++-----
 plugins/Languages/es/translation.ts               |   14 +++++++-------
 plugins/Languages/fr/translation.ts               |   20 ++++++++++----------
 plugins/Languages/hi/translation.ts               |   10 +++++-----
 plugins/Languages/id/translation.ts               |   10 +++++-----
 plugins/Languages/it/translation.ts               |   10 +++++-----
 plugins/Languages/ja/translation.ts               |   10 +++++-----
 plugins/Languages/nl/translation.ts               |   10 +++++-----
 plugins/Languages/no/translation.ts               |   10 +++++-----
 plugins/Languages/pl/translation.ts               |   10 +++++-----
 plugins/Languages/pt/translation.ts               |   10 +++++-----
 plugins/Languages/ru/translation.ts               |   10 +++++-----
 plugins/Languages/th/translation.ts               |   10 +++++-----
 plugins/Languages/tr/translation.ts               |   10 +++++-----
 plugins/Languages/zh/translation.ts               |   10 +++++-----
 plugins/Themes/Oxygen/Languages/ar/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/de/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/el/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/es/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/fr/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/hi/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/id/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/it/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/ja/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/nl/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/no/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/pl/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/pt/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/ru/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/th/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/tr/translation.ts |    4 ++--
 plugins/Themes/Oxygen/Languages/zh/translation.ts |    4 ++--
 plugins/Themes/Oxygen/interface.ui                |    2 +-
 plugins/Themes/Oxygen/options.ui                  |    2 +-
 39 files changed, 131 insertions(+), 131 deletions(-)

diff --git a/AuthPlugin.cpp b/AuthPlugin.cpp
index fcea81c..0d624d9 100644
--- a/AuthPlugin.cpp
+++ b/AuthPlugin.cpp
@@ -94,7 +94,7 @@ void AuthPlugin::run()
 								if(key==folderHash.result())
 									emit authentifiedPath(pluginBasePath);
 								else
-									ULTRACOPIER_DEBUGCONSOLE(DebugLevel_Warning,"This plugin have wrong authentification");
+									ULTRACOPIER_DEBUGCONSOLE(DebugLevel_Warning,"This plugin have wrong authentication");
 							}
 							keyDescriptor.close();
 						}
diff --git a/HelpDialog.ui b/HelpDialog.ui
index 0707ab0..e714ee8 100644
--- a/HelpDialog.ui
+++ b/HelpDialog.ui
@@ -75,7 +75,7 @@ This version is compiled as version: %1.</string>
        <item>
         <widget class="QLabel" name="label_site">
          <property name="text">
-          <string>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</string>
+          <string>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
diff --git a/SystrayIcon.cpp b/SystrayIcon.cpp
index b4fef7d..5357774 100644
--- a/SystrayIcon.cpp
+++ b/SystrayIcon.cpp
@@ -103,7 +103,7 @@ void SystrayIcon::updateSystrayIcon()
 	QString icon;
 	if(!haveListenerInfo || !havePluginLoaderInfo)
 	{
-		toolTip=tr("Searching informations...");
+		toolTip=tr("Searching information...");
 		icon="Uncaught";
 	}
 	else
diff --git a/plugins/Languages/ar/translation.ts b/plugins/Languages/ar/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/ar/translation.ts
+++ b/plugins/Languages/ar/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/de/translation.ts b/plugins/Languages/de/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/de/translation.ts
+++ b/plugins/Languages/de/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/el/translation.ts b/plugins/Languages/el/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/el/translation.ts
+++ b/plugins/Languages/el/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/es/translation.ts b/plugins/Languages/es/translation.ts
index c4ab737..15ae482 100644
--- a/plugins/Languages/es/translation.ts
+++ b/plugins/Languages/es/translation.ts
@@ -185,7 +185,7 @@ Esta versión se compila como la versión: %1.</translation>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation>Para más información consulte la página web &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</translation>
     </message>
     <message>
@@ -595,13 +595,13 @@ Esta versión se compila como la versión: %1.</translation>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
-        <translation>informations.xml no es accesible</translation>
+        <source>information.xml is not accessible</source>
+        <translation>information.xml no es accesible</translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
-        <translation>informations.xml que no se encuentran en el plug-in</translation>
+        <source>information.xml not found into the plugin</source>
+        <translation>information.xml que no se encuentran en el plug-in</translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="470"/>
@@ -651,7 +651,7 @@ Esta versión se compila como la versión: %1.</translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -796,7 +796,7 @@ No se puede cargar el contenido del plugin, por favor verifique que</translation
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation>Buscar información...</translation>
     </message>
     <message>
diff --git a/plugins/Languages/fr/translation.ts b/plugins/Languages/fr/translation.ts
index 25bf66e..a586434 100644
--- a/plugins/Languages/fr/translation.ts
+++ b/plugins/Languages/fr/translation.ts
@@ -185,8 +185,8 @@ Cette version est compilée comme version: %1.</translation>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
-        <translation>Pour plus d&apos;informations voir le site web &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</translation>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <translation>Pour plus d&apos;information voir le site web &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
@@ -595,13 +595,13 @@ Cette version est compilée comme version: %1.</translation>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
-        <translation>Le fichier informations.xml n&apos;a pas été trouvé dans le répertoire du plugin</translation>
+        <source>information.xml not found into the plugin</source>
+        <translation>Le fichier information.xml n&apos;a pas été trouvé dans le répertoire du plugin</translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
-        <translation>informations.xml n&apos;est pas accessible</translation>
+        <source>information.xml is not accessible</source>
+        <translation>information.xml n&apos;est pas accessible</translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="470"/>
@@ -666,8 +666,8 @@ Cette version est compilée comme version: %1.</translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
-        <translation>Texte anglais manquant dans le informations.xml pour la balise: %1</translation>
+        <source>English text missing into the information.xml for the tag: %1</source>
+        <translation>Texte anglais manquant dans le information.xml pour la balise: %1</translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="388"/>
@@ -791,8 +791,8 @@ depuis:%2</translation>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
-        <translation>Recherche d&apos;informations...</translation>
+        <source>Searching information...</source>
+        <translation>Recherche d&apos;information...</translation>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="120"/>
diff --git a/plugins/Languages/hi/translation.ts b/plugins/Languages/hi/translation.ts
index 6c81c6f..d704acb 100644
--- a/plugins/Languages/hi/translation.ts
+++ b/plugins/Languages/hi/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/id/translation.ts b/plugins/Languages/id/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/id/translation.ts
+++ b/plugins/Languages/id/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/it/translation.ts b/plugins/Languages/it/translation.ts
index f4702ba..aaa8ddc 100644
--- a/plugins/Languages/it/translation.ts
+++ b/plugins/Languages/it/translation.ts
@@ -174,7 +174,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation>Per maggiori dettagli vistare il sito &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</translation>
     </message>
     <message>
@@ -529,12 +529,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -585,7 +585,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -725,7 +725,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation>Cercare informazioni...</translation>
     </message>
     <message>
diff --git a/plugins/Languages/ja/translation.ts b/plugins/Languages/ja/translation.ts
index 53fa961..c485117 100644
--- a/plugins/Languages/ja/translation.ts
+++ b/plugins/Languages/ja/translation.ts
@@ -174,7 +174,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -529,12 +529,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -585,7 +585,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -725,7 +725,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/nl/translation.ts b/plugins/Languages/nl/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/nl/translation.ts
+++ b/plugins/Languages/nl/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/no/translation.ts b/plugins/Languages/no/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/no/translation.ts
+++ b/plugins/Languages/no/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/pl/translation.ts b/plugins/Languages/pl/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/pl/translation.ts
+++ b/plugins/Languages/pl/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/pt/translation.ts b/plugins/Languages/pt/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/pt/translation.ts
+++ b/plugins/Languages/pt/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/ru/translation.ts b/plugins/Languages/ru/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/ru/translation.ts
+++ b/plugins/Languages/ru/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/th/translation.ts b/plugins/Languages/th/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/th/translation.ts
+++ b/plugins/Languages/th/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/tr/translation.ts b/plugins/Languages/tr/translation.ts
index f9dff29..968bf05 100644
--- a/plugins/Languages/tr/translation.ts
+++ b/plugins/Languages/tr/translation.ts
@@ -184,7 +184,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -594,12 +594,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -650,7 +650,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -790,7 +790,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Languages/zh/translation.ts b/plugins/Languages/zh/translation.ts
index 53fa961..c485117 100644
--- a/plugins/Languages/zh/translation.ts
+++ b/plugins/Languages/zh/translation.ts
@@ -174,7 +174,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../HelpDialog.ui"/>
-        <source>For more informations see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
+        <source>For more information see the website &lt;a href=&quot;%1&quot; rel=&quot;%1&quot; title=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -529,12 +529,12 @@ This version is compiled as version: %1.</source>
     <name>PluginsManager</name>
     <message>
         <location filename="../../../PluginsManager.cpp" line="206"/>
-        <source>informations.xml is not accessible</source>
+        <source>information.xml is not accessible</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="212"/>
-        <source>informations.xml not found into the plugin</source>
+        <source>information.xml not found into the plugin</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -585,7 +585,7 @@ This version is compiled as version: %1.</source>
     </message>
     <message>
         <location filename="../../../PluginsManager.cpp" line="380"/>
-        <source>English text missing into the informations.xml for the tag: %1</source>
+        <source>English text missing into the information.xml for the tag: %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -725,7 +725,7 @@ since:%2</source>
     </message>
     <message>
         <location filename="../../../SystrayIcon.cpp" line="105"/>
-        <source>Searching informations...</source>
+        <source>Searching information...</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/plugins/Themes/Oxygen/Languages/ar/translation.ts b/plugins/Themes/Oxygen/Languages/ar/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/ar/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/ar/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/de/translation.ts b/plugins/Themes/Oxygen/Languages/de/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/de/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/de/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/el/translation.ts b/plugins/Themes/Oxygen/Languages/el/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/el/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/el/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/es/translation.ts b/plugins/Themes/Oxygen/Languages/es/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/es/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/es/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/fr/translation.ts b/plugins/Themes/Oxygen/Languages/fr/translation.ts
index 89f64bc..62d82db 100644
--- a/plugins/Themes/Oxygen/Languages/fr/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/fr/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation>Montrer la vitesse comme une information principale</translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation>Montrer la vitesse comme une information principale</translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/hi/translation.ts b/plugins/Themes/Oxygen/Languages/hi/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/hi/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/hi/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/id/translation.ts b/plugins/Themes/Oxygen/Languages/id/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/id/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/id/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/it/translation.ts b/plugins/Themes/Oxygen/Languages/it/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/it/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/it/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/ja/translation.ts b/plugins/Themes/Oxygen/Languages/ja/translation.ts
index 4ea1e31..94fb88f 100644
--- a/plugins/Themes/Oxygen/Languages/ja/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/ja/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/nl/translation.ts b/plugins/Themes/Oxygen/Languages/nl/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/nl/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/nl/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/no/translation.ts b/plugins/Themes/Oxygen/Languages/no/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/no/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/no/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/pl/translation.ts b/plugins/Themes/Oxygen/Languages/pl/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/pl/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/pl/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/pt/translation.ts b/plugins/Themes/Oxygen/Languages/pt/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/pt/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/pt/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/ru/translation.ts b/plugins/Themes/Oxygen/Languages/ru/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/ru/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/ru/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/th/translation.ts b/plugins/Themes/Oxygen/Languages/th/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/th/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/th/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/tr/translation.ts b/plugins/Themes/Oxygen/Languages/tr/translation.ts
index 43608a5..94f3990 100644
--- a/plugins/Themes/Oxygen/Languages/tr/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/tr/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/Languages/zh/translation.ts b/plugins/Themes/Oxygen/Languages/zh/translation.ts
index 4ea1e31..94fb88f 100644
--- a/plugins/Themes/Oxygen/Languages/zh/translation.ts
+++ b/plugins/Themes/Oxygen/Languages/zh/translation.ts
@@ -168,7 +168,7 @@
     </message>
     <message>
         <location filename="../../interface.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -211,7 +211,7 @@
     </message>
     <message>
         <location filename="../../options.ui"/>
-        <source>Show speed as main informations</source>
+        <source>Show speed as main information</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff --git a/plugins/Themes/Oxygen/interface.ui b/plugins/Themes/Oxygen/interface.ui
index 5f2a0d9..518082d 100644
--- a/plugins/Themes/Oxygen/interface.ui
+++ b/plugins/Themes/Oxygen/interface.ui
@@ -893,7 +893,7 @@ p, li { white-space: pre-wrap; }
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
-            <string>Show speed as main informations</string>
+            <string>Show speed as main information</string>
            </property>
           </widget>
          </item>
diff --git a/plugins/Themes/Oxygen/options.ui b/plugins/Themes/Oxygen/options.ui
index 1455e7e..aae77da 100644
--- a/plugins/Themes/Oxygen/options.ui
+++ b/plugins/Themes/Oxygen/options.ui
@@ -20,7 +20,7 @@
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
-      <string>Show speed as main informations</string>
+      <string>Show speed as main information</string>
      </property>
     </widget>
    </item>
